frontend/axios.ts

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const djangoCsrfCookieName = 'csrftoken';
+
+// Axios config for DRF requests
+axios.defaults.xsrfHeaderName = 'X-CSRFToken';
+axios.defaults.xsrfCookieName = djangoCsrfCookieName;
+axios.defaults.withCredentials = true;
+
+export default axios;
```

Comment:
Should we now import `axios` from this location every time we need it, or only when the `CSRFtoken` is needed?

Reply:
Only when the CSRF token is needed (which means anytime that we are making a POST request to our Django API, I think).